dev/tools/gen_defaults/lib/action_chip_template.dart

```diff
@@ -83,16 +83,20 @@ class _${blockName}DefaultsM3 extends ChipThemeData {
   @override
   EdgeInsetsGeometry? get padding => const EdgeInsets.all(8.0);
 
-  /// The chip at text scale 1 starts with 8px on each side and as text scaling
-  /// gets closer to 2 the label padding is linearly interpolated from 8px to 4px.
-  /// Once the widget has a text scaling of 2 or higher than the label padding
-  /// remains 4px.
+  /// The chip at effective text scale 1 starts with 8px on each side and as text
+  /// scaling gets closer to 2 the label padding is linearly interpolated from
+  /// 8px to 4px. Once the widget has a text scaling of 2 or higher than the
+  /// label padding remains 4px.
   @override
-  EdgeInsetsGeometry? get labelPadding => EdgeInsets.lerp(
-    const EdgeInsets.symmetric(horizontal: 8.0),
-    const EdgeInsets.symmetric(horizontal: 4.0),
-    clampDouble(MediaQuery.textScalerOf(context).textScaleFactor - 1.0, 0.0, 1.0),
-  )!;
+  EdgeInsetsGeometry? get labelPadding {
+    final double defaultFontSize = labelStyle?.fontSize ?? 14.0;
+    final double effectiveTextScale = MediaQuery.textScalerOf(context).scale(defaultFontSize) / 14.0;
```

Comment:
Shouldn't that be divided by the `defaultFontSize`?

Reply:
Ah I think in this case I meant to scale the padding with the font size since the threshold values seem to be empirical values that work well with the default font size, so making the font size larger should scale the padding the same way as increasing the text scale. I'll add comments in the code and update the PR description.